cmake/Config.cmake.in

```diff
@@ -0,0 +1,3 @@
+@PACKAGE_INIT@

```

Comment:
This file needs added to cmake/include.am. Fixes our “make dist” 

added: cmake/Config.cmake.in